app/models/photo_comment.rb

```diff
@@ -0,0 +1,3 @@
+class PhotoComment < ApplicationRecord
+  belongs_to :photo
```

Comment:
This has the exact behaviour of the regular comment, so it's repeated code. There should be a better solution that is more dry.